packages/tags/src/Tag.ts

```diff
@@ -90,11 +120,21 @@ export class Tag extends SizedMixin(SpectrumElement, {
         if (this.readonly) {
             return;
         }
-        this.dispatchEvent(
-            new Event('delete', {
-                bubbles: true,
-            })
-        );
+        const deleteEvent = new Event('delete', {
+            bubbles: true,
+            composed: true,
+        });
+        this.dispatchEvent(deleteEvent);
+
+        if (deleteEvent.defaultPrevented) {
+            return;
+        }
+
+        const parent = this.parentNode as HTMLElement | null;
```

Comment:
It’s possible that this causes issues when the Tag is a direct child of a shadow root. We likely want `.remove()` instead.

Reply:
Done